concepts/05 Widgets/Tabs/15 Control the Behavior.md

```diff
@@ -1,64 +1,420 @@
+---
 
 As an alternative, you can use the [selectedItem](/api-reference/10%20UI%20Widgets/CollectionWidget/1%20Configuration/selectedItem.md '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/#selectedItem') (for *"single"* **selectionMode**) or [selectedItems](/api-reference/10%20UI%20Widgets/dxTabs/1%20Configuration/selectedItems.md '/Documentation/ApiReference/UI_Widgets/dxTabs/Configuration/#selectedItems') (for *"multiple"* **selectionMode**) options.
 
+---
+##### jQuery
+
+    <!--HTML-->
+    <div id="tabsContainer"></div>
+
     <!--JavaScript-->
     var tabs = [
-        { text: "User", icon: "user" },
-        { text: "Find", icon: "find" },
-        { text: "Favorites", icon: "favorites" }
+        { text: "User" },
+        { text: "Comment" },
+        // ...
     ];
 
     $(function() {
         $("#tabsContainer").dxTabs({
-            items: tabs,
-            selectedItem: tabs[1],
-            // === or ===
+            dataSource: tabs,
             selectionMode: 'multiple',
-            selectedItems: [ tabs[1], tabs[2] ]
+            selectedItems: [ tabs[0], tabs[1] ]
         });
     });
 
+##### Angular
+
+    <!--HTML-->
+    <dx-tabs
+        [dataSource]="tabs"
+        selectionMode="multiple"
+        [selectedItems]="[tabs[0], tabs[1]]">
+    </dx-tabs>
+
+    <!--TypeScript-->
+    import { DxTabsModule } from "devextreme-angular";
+    // ...
+    export class AppComponent {
+        tabs = [
+            { text: "User" },
+            { text: "Comment" },
+            // ...
+        ];
+    }
+    @NgModule({
+        imports: [
+            // ...
+            DxTabsModule
+        ],
+        // ...
+    })
+
+##### Vue
+
+    <!--tab: App.vue-->
+    <template>
+        <DxTabs
+            :items="tabs"
+            :selection-mode="multiple"
+            :selected-items="[tabs[0], tabs[1]]" /> 
+    </template>
+    <script>
+    import DxTabs from "devextreme-vue/tabs";
+
+    export default {
+        components: {
+            DxTabs
+        },
+        data() {
+            return {
+                tabs: [
+                    { text: "User" },
+                    { text: "Comment" },
+                    // ...
+                ]
+            };
+        }
+    };
+    </script>
+
+##### React
+
+    <!--tab: App.js-->
+    import React from 'react';
+    import { Tabs } from 'devextreme-react/tabs';
+
+    class App extends React.Component {
+        constructor() {
+            this.tabs = [
+                { text: "User" },
+                { text: "Comment" },
+                // ...
+            ];
+            this.state = {
+                selectedItems: [this.tabs[0], this.tabs[1]]
+            }
+        }
+
+        render() {
+            return (
+                <Tabs
+                    items={this.tabs}
+                    selectionMode="multiple",
+                    selectedItems={this.state.selectedItems}
```

Comment:
If you are reading the state, you should implement changing it. Try to handle `onOptionChanged` ([here is a similar case](https://js.devexpress.com/Documentation/20_1/Guide/Widgets/DataGrid/Selection/#API/Clear_Selection_Settings)) or declare `selectedItems` as `defaultSelectedItems`.